frontend/utils/distribution/group-income-distribution.test.js

```diff
@@ -15,33 +55,22 @@ describe('Chunk 0: Adjustment Tests',
       // u2: Income 10$ (a.k.a needs $2)
       // Expected: [{ amount: 2, from: 'u1', to: 'u2' }]
       function () {
-        const dist = groupIncomeDistributionAdjustFirstLogic({
-          getters: {
-            'groupProfiles': {
-              'u1': {
-                'incomeDetailsType': 'pledgeAmount',
-                'pledgeAmount': 10
-              },
-              'u2': {
-                'incomeDetailsType': 'incomeAmount',
-                'incomeAmount': 10
-              }
-            },
-            'currentGroupState': {
-              'payments': {}
-            },
-            'monthlyPayments': {
-              '2020-11': {
-                'paymentsFrom': {}
-              }
+        const dist = groupIncomeDistributionAdjustFirstLogic(unfoldParameters({
+          'profiles': {
+            'u1': {
+              'incomeDetailsType': 'pledgeAmount',
+              'pledgeAmount': 10
             },
-            'groupSettings': {
-              'mincomeAmount': 12
+            'u2': {
+              'incomeDetailsType': 'incomeAmount',
+              'incomeAmount': 10
             }
           },
-          monthstamp: '2020-11',
-          adjusted: true
-        })
+          'payments': [],
+          'mincome': 12,
+          'monthstamp': '2020-11',
+          'adjusted': true
```

Comment:
I think this can be simplified to make it even more readable. Here are some suggestions:

- Use defaults so that you can leave out data. For example, if the `payments` is empty (as it is here), don't require it to be included. If the `monthstamp` is always the same for the purpose of the tests, fill it in with a default value so that it too doesn't need to be specified. If `adjusted: true` is the value for that most of the time, then have that be the default and only specify it when it changes. DRY!
- Anywhere you have things spread across multiple lines, where they could be specified on a single line within 70 columns width, make them a single line (e.g. `u2: { 'incomeDetailsType': 'incomeAmount', 'incomeAmount': 10 }`)
- Anywhere something can be rewritten in a simpler way, do it (e.g. `u2: { need: 2 }` instead of the previous version)

Then these tests will read like a children's book and will be both super easy to write and read and understand.